### PR TITLE
Amend "Preventing importing of courses with invalid catalog numbers"

### LINKED ISSUE #89: Show reading courses
Solves #8 by showing those courses - they're reading courses, and it would be great to see them. I'll make a PR.

### PR DESCRIPTION
**Pull request by awonnacott**
This amends commit d90b0a3334b96268c0f30745bc257c5e81737f8f.

Handles #8 correctly

Closes #89 
  
Reading courses are valid courses with two-digit course numbers between 90 and 99.

### CODE REVIEW

**Comment by sebthedev** [comment]
Thanks for the PR, @awonnacott. I’ll review when I get back from vacation.

File: importers/importBasicCourseDetails.js [hunk 1]
```diff
@@ -116,10 +116,6 @@ var importSubject = function (semester, subject) {
       process.stdout.write(' ' + courseData.catalog_number)
     }
 
-    if (typeof (courseData.catalog_number) === 'undefined' || courseData.catalog_number.length < 3) {
```
Comment (sebthedev) [inline importers/importBasicCourseDetails.js]: I understand what you're doing here, @awonnacott, but we still need error-handling code to catch cases where the catalog number _is_ undefined or is shorter than three characters and not 90.

Could you re-submit with a conditional something along the lines of:
```javascript
if (typeof (courseData.catalog_number) === 'undefined' || courseData.catalog_number.length < 3 && courseData.catalog_number != 90)
```

I would do it myself, but I want to make sure you get credit for your contribution!

Comment (awonnacott) [reply importers/importBasicCourseDetails.js]: Sure!

Comment (awonnacott) [reply importers/importBasicCourseDetails.js]: How does it look now?

Comment (awonnacott) [reply importers/importBasicCourseDetails.js]: bump

**Comment by sebthedev** [comment]
Thanks for the contribution, @awonnacott. Your changes to the importer script will be run within the next 24 hours (it runs automatically).

**Pull request merged**